File: PostBox/src/main/java/com/mvp/bean/Box5.java
package com.mvp.bean;

public class Box5 extends BaseBox {

    private final float length = 39.5f;
    private final float width = 27.5f;
    private final float height = 23f;

    public Box5(float length, float width, float height) {
        super(length, width, height);
    }

    @Override
    public boolean validate() {
        float totalOfBox5 = length + width + height;
        float inputOfBox5 = getLength() + getWidth() + getHeight();

        if (getLength() > length || getWidth() > width || getHeight() > height)
            return false;
        else if (inputOfBox5 <= totalOfBox5) {
            return true;
        } else
            return true;
    }

}

File: PostBox/src/main/kotlin/com/mvp/bean/Box5.kt
package com.mvp.bean.kt

class Box5(length: Float?, width: Float?, height: Float?) : BaseBox(length, width, height) {

    init {
        // property define in constructor without val can only access in init block
        println("set box5 length to base box, length = $length")
        println("set box5 width to base box, width = $width")
        println("set box5 height to base box, height = $height")
    }

}